src/common/kafka/AR52Serializer.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2023 European Spallation Source, ERIC. see LICENSE file
+//===----------------------------------------------------------------------===//
+///
+/// \file
+///
+/// \brief flatbuffer serialization into ar52 schema
+///
+/// See https://github.com/ess-dmsc/streaming-data-types
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "Producer.h"
+#include "ar52_readout_data_generated.h"
+
+
+class AR52Serializer {
```

Comment:
Yes, it will be used to stream a fraction of the raw readout data to kafka, say 2 out of every 1000 packets. This data can then be used by the detector group to subscribe to for histogramming or specialised plotting. It was discussed about three weekss ago. I don't think we have been in the office together since then ;-)

Reply:
I've updated the readme table with a short description.

Reply:
Thanks for the update on the description.